pkg/storaged/storage-controls.jsx

```diff
@@ -257,14 +258,16 @@ export class StorageUsageBar extends React.Component {
         var stats = this.props.stats;
         var fraction = stats ? stats[0] / stats[1] : null;
 
-        return (
-            <div className="progress">
-                { stats
-                    ? <div className={ "progress-bar" + (fraction > this.props.critical ? " progress-bar-danger" : "") }
-                        style={{ width: fraction * 100 + "%" }} />
-                    : null
-                }
-            </div>
-        );
+        if (stats) {
+            return (
+                <Progress value={stats[0]} max={stats[1]}
+                    valueText={utils.format_fsys_usage(stats[0], stats[1])}
+                    label={utils.format_fsys_usage(stats[0], stats[1])}
```

Comment:
Do we need both `valueText` and `label`?  If so, let's call `format_fsys_usage` only once, to avoid duplicating code.

Reply:
Yes we need both:
From PF4 docs:
```
valueText | string |   | null as string | Accessible text description of current progress value, for when value is not a percentage. Use with label.
-- | -- | -- | -- | --


```

Reply:
Fixed.